src/LinuxTracingIntegrationTests/LinuxTracingIntegrationTest.cpp

```diff
@@ -438,6 +443,7 @@ void VerifyOrderOfAllEvents(const std::vector<orbit_grpc_protos::ProducerCapture
         EXPECT_GE(event.thread_state_slice().end_timestamp_ns(), previous_event_timestamp_ns);
         previous_event_timestamp_ns = event.thread_state_slice().end_timestamp_ns();
         break;
+      case orbit_grpc_protos::ProducerCaptureEvent::kTracepointCallstack:
```

Comment:
I would prefer an explicit `ORBIT_UNREACHABLE();` instead of the fallthrough (like it's done in the rest of the switch).